### PR TITLE
[Fix][WRS-1658] Add PDF export (out of experimental)

### PR DESCRIPTION
**Pull request by brapoprod**
This PR adds PDF as the export setting

### CODE REVIEW

**Review by abdelhalimkhouas — DISMISSED** [review]

File: src/utils/documentExportHelper.ts [hunk 1]
```diff
@@ -44,22 +51,32 @@ export const getDownloadLink = async (
...
+            pdfBody.engineVersion = `${engineVersion}${engineCommitSha && `-${engineCommitSha}`}`;
         }
+
         const config: HttpHeaders = {
             method: 'POST',
             headers: {
                 'Content-Type': 'application/json',
             },
-            body: (documentResponse.data as string) ?? null,
+            body: format !== DownloadFormats.PDF ? (documentResponse.data as string) ?? null : JSON.stringify(pdfBody),
```
Comment (brapoprod) [inline src/utils/documentExportHelper.ts:68]: for this one I'm not sure what it actually does

**Review by srares76 — APPROVED** [review]

File: src/utils/documentExportHelper.ts (hunk 1 above)
Comment (abdelhalimkhouas) [reply src/utils/documentExportHelper.ts:68]: Seems like with `PDF` you can export multiple layouts as you can see `layoutsToExport` so the body when exporting `PDF` is slightly different we add the `layoutsToExport` with the list of layouts ids

**Review by abdelhalimkhouas — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     },
     "dependencies": {
         "@chili-publish/grafx-shared-components": "^0.56.0",
-        "@chili-publish/studio-sdk": "1.7.4",
+        "@chili-publish/studio-sdk": "1.7.10",
```
Comment (psamusev) [inline package.json:28]: there is more recent version then this one. Should we switch to it?

Comment (brapoprod) [reply package.json]: sure!

**Review by abdelhalimkhouas — APPROVED** [review]

File: src/utils/documentExportHelper.ts (hunk 1 above)
Comment (brapoprod) [reply src/utils/documentExportHelper.ts:68]: no I'm specifically talking about the body in config, never saw it before

**Review by psamusev — APPROVED** [review]

**Comment by brapoprod** [comment]
Approved by aqa

**Pull request merged**